README.md

```diff
@@ -5,7 +5,7 @@ Node.js
 
 Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js
 uses an event-driven, non-blocking I/O model that makes it lightweight and
-efficient. The Node.js package ecosystem, npm, is the largest ecosystem of open
+efficient. The Node.js package ecosystem, [npm](https://www.npmjs.com), is the largest ecosystem of open
```

Comment:
Eh, I'm not too thrilled about this link. The frontpage of npmjs.com is probably not the best place to link a newcomer to. Probably best to leave out that link if we can't find a better target.